korge/src/commonMain/kotlin/korlibs/korge/component/StageComponent.kt

```diff
@@ -36,26 +36,28 @@ fun <T : View> T.onAttachDetach(onAttach: Views.(T) -> Unit = {}, onDetach: View
     return this
 }
 
+private val viewsToTrack = linkedSetOf<View>()
```

Comment:
This should not be done. We might have multiple korge instances. We cannot have globals. You can either add it to the Views instance as a property, or as an extra propery. Ideally extra property as this is an external component. You can retrieve the instance once at the beginning of the `registerStageComponent` function